### PR TITLE
add riotux v1.0.2 with npm auto-update config

### LINKED ISSUE #7564: [REQUEST] riotux
**Library name:** riotux
**Git repository url:** https://github.com/luisvinicius167/riotux
**npm package url(optional):**  https://www.npmjs.com/package/riotux
**License(s):** MIT
**Official homepage:** http://luisvinicius167.github.io/riotux/

### PR DESCRIPTION
**Pull request by nolsherry**
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/luisvinicius167/riotux
 * Official website (optional, not the repository): https://luisvinicius167.github.io/riotux/
 * NPM package url (optional): https://www.npmjs.com/package/riotux
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 34
   - Count of watchers: 3
   - Count of forks: 1
 * NPM download stats (optional):
   - Downloads in the last day: 49
   - Downloads in the last week: 62
   - Downloads in the last month: 169

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [X] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [X] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [X] Has valid tags for each versions (for git auto-update)
 * [X] Auto-update setup
 * [X] Auto-update target/source is valid.
 * [X] Auto-update filemap is correct.

# Git commit checklist
 * [X] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [ ] The parent of the commit(s) in the PR is not old than 3 days.
 * [X] Pull request is sending from a non-master branch with meaningful name.
 * [ ] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [X] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

close #7564, cc @luisvinicius167

### CODE REVIEW

**Comment by nolsherry** [comment]
@dannyxx001, please help me to review this pull request, thanks!

File: ajax/libs/riotux/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "riotux",
+  "filename": "riotux.min.js",
+  "npmName": "riotux",
+  "version": "1.0.2",
+  "description": "Simple Event Controller for Riot.js, inspired in Flux Pattern.",
+  "npmFileMap": [
```
Comment (PeterDaveHello) [inline ajax/libs/riotux/package.json]: Please put `npmName` & `npmFileMap` together.

File: ajax/libs/riotux/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }
```
Comment (PeterDaveHello) [inline ajax/libs/riotux/package.json]: Add one more map here:
```js
    {
      "basePath": "",
      "files": [
        "riotux.js"
      ]
    }
```

**Comment by nolsherry** [comment]
@dannyxx001,  please help me to review this pull request, thanks!

**Comment by PeterDaveHello** [comment]
@nolsherry you should rebase your branch on the latest master branch next time.

**Pull request merged**